filters/auth/grant_test.go

```diff
@@ -21,10 +21,14 @@ import (
 const (
 	testToken                = auth.TestToken
 	testRefreshToken         = auth.TestRefreshToken
+	testAccessTokenExpiresIn = auth.TestAccessTokenExpiresIn
+	testClientID             = "some-id"
+	testClientSecret         = "some-secret"
 	testAccessCode           = "quxquuxquz"
 	testSecretFile           = "testdata/authsecret"
-	testAccessTokenExpiresIn = int(time.Hour / time.Second)
```

Comment:
IMO should be time.Duration - there are many conversions to time.Duration and only one place where int is required.

Reply:
✅  Done